ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -132,21 +153,29 @@ def aws_cleanup():
         help="The name of the AWS region to delete the resources from"
     )
     parser.add_argument(
-        '--force',
-        action='store_true',
+        '--prefix',
+        action='append',
         required=False,
-        help="Force cluster cleanup. "
-             "User will not be prompted for confirmation. "
-             "WARNING: this utility is destructive, only use this option if "
-             "you know what you are doing."
+        type=prefix_hour_mapping,
+        help="""
+            Additional prefix:hour combo to treat as a special rule.
+            Clusters starting with this prefix will only be cleaned up if
+            their runtime exceeds the provided hour(this takes precedence
+            over the value provided to --hours).
+            Example: --prefix foo:24 --prefix bar:48
+            """
```

Comment:
I would mention the `never` keyword also here in the help for the `--prefix` argument.

Reply:
Done